samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java

```diff
@@ -35,9 +35,16 @@
     private static CorfuRuntime getRuntimeAndConnect(String configurationString) {
 
         return CorfuRuntime.fromParameters(CorfuRuntime.CorfuRuntimeParameters.builder()
-                .build())
-                .parseConfigurationString(configurationString)
-                .connect();
+            .useFastLoader(false)
+            .cacheDisabled(true)
+            .tlsEnabled(true)
+            .keyStore("/config/cluster-manager/corfu/private/keystore.jks")
+            .ksPasswordFile("/config/cluster-manager/corfu/private/keystore.password")
+            .trustStore("/config/cluster-manager/corfu/public/truststore.jks")
+            .tsPasswordFile("/config/cluster-manager/corfu/public/truststore.password")
```

Comment:
Why the files are using these paths instead of a local file in the resource folder?

Reply:
Reverting to not using these parameters.

Doing so will not allow sample app to use tls but that is independent of browser changes.  So can be addressed in a different PR.